Add and update Channels

Alstruit opened the pull request:
Need to resolve PR iptv-org/database#644 before merging.

Resolves #9207, resolves #9208, resolves #9209, resolves #9210, resolves #9211.
Resolves #2577, resolves #7912, resolves #8006, resolves #8698, resolves #8979, resolves #9134, resolves #9200, resolves #9218, resolves #9220, resolves #9221, resolves #9225, resolves #9226, resolves #9227, resolves #9228, resolves #9229, resolves #9235, resolves #9236, resolves #9240, resolves #9242, resolves #9243, resolves #9245, resolves #9246, resolves #9247, resolves #9249, resolves #9250, resolves #9251, resolves #9252, resolves #9253, resolves #9254, resolves #9255, resolves #9256, resolves #9257, resolves #9258, resolves #9260, resolves #9261, resolves #9262, resolves #9263, resolves #9267, resolves #9268, resolves #9269, resolves #9270, resolves #9271, resolves #9272, resolves #9274, resolves #9275, resolves #9282, resolves #9285, resolves #9286, resolves #9287, resolves #9288, resolves #9290, resolves #9291, resolves #9292, resolves #9294, resolves #9298, resolves #9299, resolves #9301, resolves #9304, resolves #9305, resolves #9306, resolves #9307, resolves #9308, resolves #9309, resolves #9313, resolves #9322, resolves #9325, resolves #9326, resolves #9327, resolves #9329, resolves #9330, resolves #9331, resolves #9332, resolves #9334, resolves #9335, resolves #9337, resolves #9340, resolves #9341, resolves #9342, resolves #9343, resolves #9344, resolves #9345, resolves #9346, resolves #9347, resolves #9349, resolves #9352, resolves #9354, resolves #9356, resolves #9357, resolves #9358, resolves #9359, resolves #9360, resolves #9361, resolves #9362, closes #9374.
Resolves #6035, resolves #5436, resolves #4132, resolves #5617.

LaneSh4d0w commented:
Dang, you sure do the job right. Going to check the different URLs for anything we could change.

LaneSh4d0w commented on streams/us.m3u:
```diff
@@ -1947,3 +1947,225 @@ https://2-fss-2.streamhoster.com/pl_120/amlst:203530-1426406/playlist.m3u8
+#EXTINF:-1 tvg-id="GodsLearningChannel.us" status="",GLC TV God's Learning Channel (480p)
+https://2-fss-1.streamhoster.com/pl_122/200562-1176456-1/playlist.m3u8
+#EXTINF:-1 tvg-id="GlobalBuddhistNetwork.us" status="",GBN Global Buddhist Network (1080p)
+https://us.gbnus.com/live/dhamma_high/playlist.m3u8
```
Going to add this one over IPTV_Exception, as this one contains different qualities but no adaptive stream.

LaneSh4d0w replied:
Please check https://github.com/LaneSh4d0w/IPTV_Exception/raw/master/channels/us/gbnus.m3u8 and tell me if you can see both High and Low.

Alstruit replied:
> Please check https://github.com/LaneSh4d0w/IPTV_Exception/raw/master/channels/us/gbnus.m3u8 and tell me if you can see both High and Low.

Yes I can see both high and low streams.

LaneSh4d0w replied:
Perfect, then
```suggestion
https://github.com/LaneSh4d0w/IPTV_Exception/raw/master/channels/us/gbnus.m3u8
```

Alstruit committed:
Preliminary

Subject to change
To put on PR: #9207,#9208,#9209,#9210,#9211
Apply adaptive stream from @LaneSh4d0w.
Reduce scope for PR#9216

AntiPontifex commented:
First of all, thank you for your efforts. I have noticed, however, that the Marin Community Channels have been duplicated. Originally, there were three distinct channels: hope this helps.

Eyob111 commented:
guys, can you add more Ethiopian channels that are free on streaming websites? 
like -  Fana, Amhara ,ebs tv.

Alstruit committed:
Fix incorrect Marin TV links

Alstruit commented:
> First of all, thank you for your efforts. I have noticed, however, that the Marin Community Channels have been duplicated. Originally, there were three distinct channels: hope this helps.

I don't know what do you mean duplicated, but I did notice I missed one channel and one of the channels is incorrectly linked.

Alstruit committed:
Update/Remove livestream and vimeo links

Remove duplicates.
Convert into https links.
Add a few Geo-blocked links.
Reopen issue #8938. Previous issue #7506.

Alstruit committed:
Add and update channels.

Add channels
Remove similar duplicates
Fix missing IDs
Update out of date links
Convert some youtube links into Youtube Channel ID.
Remove duplicates

Alstruit committed:
Update br.m3u and us.m3u. Add td.m3u

Fix missing resolutions.

Alstruit committed:
Fix npx. Add channels.

Reference PR #9232.

Alstruit committed:
Merge branch 'master' into alstruit-10_w1

Fix broken link.

Alstruit committed:
Merge branch 'master' into alstruit-10_w1

Fix incorrect link placement

Alstruit committed:
Merge branch 'master' into alstruit-10_w1
